bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -1611,7 +1664,8 @@ void DataAggregator::processBranchEvents() {
   for (const auto &AggrLBR : BranchLBRs) {
     const Trace &Loc = AggrLBR.first;
     const TakenBranchInfo &Info = AggrLBR.second;
-    doBranch(Loc.From, Loc.To, Info.TakenCount, Info.MispredCount);
+    doBranch(Loc.From, Loc.To, Info.TakenCount, Info.MispredCount,
+             /*IsPreagg=*/false);
```

Comment:
nit (I believe that's LLVM's preferred way):
```suggestion
             /*IsPreagg*/ false);
```